### PR TITLE
Add resistance rule element to Fire Resistance feat

### PR DESCRIPTION
**Pull request by KSops**
Add missing RE for Fire Resistance feat
https://2e.aonprd.com/Feats.aspx?ID=2973

### CODE REVIEW

File: packs/data/feats.db/fire-resistance.json [hunk 1]
```diff
@@ -34,6 +34,11 @@
                 "selector": "saving-throw",
                 "type": "circumstance",
                 "value": 1
+            },
+            {
+                "key": "Resistance",
+                "type": "fire",
+                "value": "max(1,floor(@actor.level/2))"
```
Comment (stwlam) [inline packs/data/feats.db/fire-resistance.json:41]: The `max` shouldn't be there

**Review by stwlam — CHANGES_REQUESTED** [review]

File: packs/data/feats.db/fire-resistance.json (hunk 1 above)
Comment (TikaelSol) [inline packs/data/feats.db/fire-resistance.json:41]:
```suggestion
                "value": "floor(@actor.level/2)"
```

**Comment by KSops** [comment]
Wasn't sure if the max was some convention or not based on other existing REs like packs/data/feats.db/nothing-but-fluff.json
Thanks for tweaking

**Comment by stwlam** [comment]
> Wasn't sure if the max was some convention or not based on other existing REs like packs/data/feats.db/nothing-but-fluff.json Thanks for tweaking

Some abilities and effects set a minimum of 1, but this one doesn't.

**Pull request merged**